packages/app/src/background/services/credentials/utils.ts

```diff
@@ -154,85 +150,86 @@ export async function deserializeCryptkeeperVerifiableCredential(
  */
-export async function deserializeVerifiableCredential(
-  serializedVerifiableCredential: string,
-): Promise<IVerifiableCredential> {
-  if (!serializedVerifiableCredential) {
+export async function deserializeVC(serializedVC: string): Promise<IVerifiableCredential> {
+  if (!serializedVC) {
     throw new Error("Serialized Verifiable Credential is not provided");
   }
 
-  const deserializedVerifiableCredential = JSON.parse(serializedVerifiableCredential) as IVerifiableCredential;
+  const deserializedVC = JSON.parse(serializedVC) as IVerifiableCredential;
 
-  return verifiableCredentialSchema.validate(deserializedVerifiableCredential);
+  return verifiableCredentialSchema.validate(deserializedVC);
 }
 
 /**
  * Serializes a VerifiablePresentation object into a JSON string.
  * @param verifiablePresentation An object representing a VerifiablePresentation.
  * @returns A string representing a VerifiablePresentation.
  */
-export function serializeVerifiablePresentation(verifiablePresentation: IVerifiablePresentation): string {
+export function serializeVP(verifiablePresentation: IVerifiablePresentation): string {
   if (!verifiablePresentation.verifiableCredential) {
     return stringify(verifiablePresentation);
   }
 
-  const serializedVerifiableCredentials = verifiablePresentation.verifiableCredential.map((verifiableCredential) =>
-    serializeVerifiableCredential(verifiableCredential),
+  const serializedVCs = verifiablePresentation.verifiableCredential.map((verifiableCredential) =>
+    serializeVC(verifiableCredential),
   );
 
   return stringify({
     ...verifiablePresentation,
-    verifiableCredential: serializedVerifiableCredentials,
+    verifiableCredential: serializedVCs,
   });
 }
 
 /**
  * Determines if a string represents a valid VerifiableCredential.
- * @param serializedVerifiableCredential An string representing a VerifiableCredential.
+ * @param serializedVC An string representing a VerifiableCredential.
  * @returns The string if it is a valid VerifiableCredential, otherwise throws an error.
  */
-export async function validateSerializedVerifiableCredential(serializedVerifiableCredential: string): Promise<string> {
-  await deserializeVerifiableCredential(serializedVerifiableCredential);
+export async function validateSerializedVC(serializedVC: string): Promise<string> {
+  await deserializeVC(serializedVC);
 
-  return serializedVerifiableCredential;
+  return serializedVC;
 }
 
 /**
  * Generates a hash of a VerifiableCredential.
  * @param verifiableCredential An object representing a VerifiableCredential.
  * @returns A string representing the hash of the VerifiableCredential.
  */
-export function hashVerifiableCredential(verifiableCredential: IVerifiableCredential): string {
-  return SHA256(serializeVerifiableCredential(verifiableCredential)).toString();
+export function hashVC(verifiableCredential: IVerifiableCredential): string {
```

Comment:
```suggestion
export function hashVC(vc: IVC): string {
```

Reply:
I think IVerifiableCredential is a good name for interface as well as IVerifiablePresentation.

Reply:
I agree. These two interfaces are important and we should use full names.